cmd/loadTests.go

```diff
@@ -78,32 +93,41 @@ type ErrorCount struct {
-	AverageTimeToRunPipelineFailed    float64           `json:"runPipelineFailedTimeAvg"`
-	UserCreationFailureCount          int64             `json:"createUserFailures"`
-	UserCreationFailureRate           float64           `json:"createUserFailureRate"`
-	ResourceCreationFailureCount      int64             `json:"createResourcesFailures"`
-	ResourceCreationFailureRate       float64           `json:"createResourcesFailureRate"`
-	PipelineRunFailureCount           int64             `json:"runPipelineFailures"`
-	PipelineRunFailureRate            float64           `json:"runPipelineFailureRate"`
-	ErrorCounts                       []ErrorCount      `json:"errorCounts"`
-	Errors                            []ErrorOccurrence `json:"errors"`
-	ErrorsTotal                       int               `json:"errorsTotal"`
+	Timestamp                         string  `json:"timestamp"`
+	EndTimestamp                      string  `json:"endTimestamp"`
+	MachineName                       string  `json:"machineName"`
+	BinaryDetails                     string  `json:"binaryDetails"`
+	ComponentRepoUrl                  string  `json:"componentRepoUrl"`
+	NumberOfThreads                   int     `json:"threads"`
+	NumberOfUsersPerThread            int     `json:"usersPerThread"`
+	NumberOfUsers                     int     `json:"totalUsers"`
+	PipelineSkipInitialChecks         bool    `json:"pipelineSkipInitialChecks"`
+	LoadTestCompletionStatus          string  `json:"status"`
+	AverageTimeToSpinUpUsers          float64 `json:"createUserTimeAvg"`
+	MaxTimeToSpinUpUsers              float64 `json:"createUserTimeMax"`
+	AverageTimeToCreateResources      float64 `json:"createResourcesTimeAvg"`
+	MaxTimeToCreateResources          float64 `json:"createResourcesTimeMax"`
+	AverageTimeToRunPipelineSucceeded float64 `json:"runPipelineSucceededTimeAvg"`
+	MaxTimeToRunPipelineSucceeded     float64 `json:"runPipelineSucceededTimeMax"`
+	AverageTimeToRunPipelineFailed    float64 `json:"runPipelineFailedTimeAvg"`
+
+	AverageTimeToDeploymentSucceeded float64 `json:"DeploymentSucceededTimeAvg"`
+	MaxTimeToDeploymentSucceeded     float64 `json:"DeploymentSucceededTimeMax"`
+	AverageTimeToDeploymentFailed    float64 `json:"DeploymentFailedTimeAvg"`
+
+	UserCreationFailureCount     int64   `json:"createUserFailures"`
+	UserCreationFailureRate      float64 `json:"createUserFailureRate"`
+	ResourceCreationFailureCount int64   `json:"createResourcesFailures"`
+	ResourceCreationFailureRate  float64 `json:"createResourcesFailureRate"`
+	PipelineRunFailureCount      int64   `json:"runPipelineFailures"`
+	PipelineRunFailureRate       float64 `json:"runPipelineFailureRate"`
+
+	DeploymentFailureCount int64   `json:"DeploymentFailures"`
+	DeploymentFailureRate  float64 `json:"DeploymentFailureRate"`
+
+	ErrorCounts []ErrorCount      `json:"errorCounts"`
+	Errors      []ErrorOccurrence `json:"errors"`
+	ErrorsTotal int               `json:"errorsTotal"`
```

Comment:
Hello! I assume this only adds a line or two, but diff is huge. Also above. Could the whitespace changes be reverted?

Reply:
Hi, It adds more lines

Reply:
I will try to remove whitespace